lxd/storage/backend_lxd.go

```diff
@@ -2338,6 +2338,11 @@ func (b *lxdBackend) CreateInstanceSnapshot(inst instance.Instance, src instance
 	// There's no need to pass config as it's not needed when creating volume snapshots.
 	vol := b.GetVolume(volType, contentType, volStorageName, nil)
 
+	// Lock this operation to ensure that the only one snapshot is made at the time.
+	// Other operations will wait for this one to finish.
+	unlock := locking.Lock(drivers.OperationLockName("CreateInstanceSnapshot", b.name, volType, contentType, src.Name()))
```

Comment:
Please use `vol.Type()` here for consistency with the equivalent line for custom volumes.

Reply:
Fixed